spec/lib/dispatch-rider/subscriber_spec.rb

```diff
@@ -93,10 +93,10 @@ def process(options)
       context "when process terminates" do
         before do
           stub_const("Terminator", Class.new(DispatchRider::Handlers::Base) {
-            def process(options)
-              Process.kill("TERM", 0)
-            end
-          })
+                                   def process(options)
```

Comment:
Unused method argument - `options`. If it's necessary, use `_` or `_options` as an argument name to indicate that it won't be used. You can also write as `process(*)` if you want the method to accept any arguments but don't care about them.